server/.eslintrc.yml

```diff
@@ -18,3 +18,22 @@ rules:
       - info
   prefer-promise-reject-errors:
     - off
+  import/no-import-module-exports:
+    - off
+  import/no-relative-packages:
+    - off
```

Comment:
let's try avoid `off` and rewrite or turn off only in places where we really need it